RELEASE.md

```diff
@@ -19,6 +19,7 @@
 * Fix bug where `micropkg` manifest section in `pyproject.toml` isn't recognised as allowed configuration.
```

Comment:
```suggestion
* Fixed bug where `micropkg` manifest section in `pyproject.toml` isn't recognised as allowed configuration.
```

Reply:
Fixed in [78f2f18](https://github.com/kedro-org/kedro/pull/2159/commits/78f2f18df0e1b817af1399a87ed0665a5e6d2ed5).